xls/integrator/ir_integrator.cc

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License
+
+#include "xls/integrator/ir_integrator.h"
+
+#include "xls/ir/ir_parser.h"
+
+namespace xls {
+
+// Return the name of 'function', prepended with the 'package' name.
+std::string GetParsableQualifiedFunctionName(Package* package,
+                                             const Function* function) {
+  return "PKGzzz" + package->name() + "zzzFNzzz" + function->name();
```

Comment:
PKGzzz? If you need to generate unique_names, check out xls/ir/name_uniquer.

Reply:
Done.